posthog/api/session_recording.py

```diff
@@ -115,7 +115,9 @@ def retrieve(self, request: request.Request, *args: Any, **kwargs: Any) -> respo
         distinct_id = session_recording_meta_data["distinct_id"]
 
         try:
-            person: Union[Person, None] = Person.objects.get(persondistinctid__distinct_id=distinct_id, team=self.team)
+            person: Union[Person, None] = Person.objects.get(
+                persondistinctid__distinct_id=distinct_id, persondistinctid__team_id=self.team, team=self.team
```

Comment:
Do you think this is a speed up because earlier this query wasn't using the index on PersonDistinctId, and now it will? Or is there some other reason?

Reply:
Yeah, exactly. It just adds a `AND "posthog_persondistinctid"."team_id" = %(team_id)s` to the query, so it takes advantage of the index.

When I run the 2 queries in metabase, it changes the mean run time from ~19s to 0.17s.

Reply:
Yeah sounds good!